### PR TITLE
Cleaning up & formatting

### PR DESCRIPTION
**Pull request by Sparkymod**
- Removing some files and folder that are not need it.
- Cleaning up excess of line and formatting the project to the new standards.

### CODE REVIEW

File: MapleServer2/PacketHandlers/Game/StateSkillHandler.cs [hunk 1]
```diff
@@ -3,25 +3,29 @@
...
-        public override void Handle(GameSession session, PacketReader packet) {
+        public override void Handle(GameSession session, PacketReader packet)
+        {
             byte function = packet.ReadByte();
-            if (function == 0) {
+            if (function == 0)
+            {
                 // This count seems to increase for each skill used
-                int counter = packet.ReadInt();
+                packet.ReadInt();   // Counter
```
Comment (AlanMorel) [inline MapleServer2/PacketHandlers/Game/StateSkillHandler.cs:20]: Why remove the variable declaration just to add the variable name as a comment next to it?

File: MapleServer2/Types/FieldState.cs [hunk 2]
```diff
@@ -1,72 +1,76 @@
...
+        public void AddPortal(IFieldObject<Portal> portal)
+        {
             portals[portal.ObjectId] = portal;
         }
 
-        public bool RemovePortal(int objectId) {
-            return portals.Remove(objectId, out _);
-        }
-
-        public void AddMob(IFieldObject<Mob> mob)
```
Comment (AlanMorel) [inline MapleServer2/Types/FieldState.cs:67]: Why was this removed?

File: MapleServer2/Types/Mob.cs [hunk 3]
```diff
@@ -1,19 +0,0 @@
...
-        public byte Animation;
-        public PlayerStats stats;
-
-        public Mob(int id)
-        {
-            this.Id = id;
-            this.Animation = 255;
-        }
-    }
-}
```
Comment (AlanMorel) [inline MapleServer2/Types/Mob.cs:19]: I do not understand why we are deleting things that we will eventually need, this makes zero sense.

File: MapleServer2/PacketHandlers/Game/UserChatHandler.cs [hunk 4]
```diff
@@ -5,21 +5,25 @@
...
         public UserChatHandler(ILogger<GamePacketHandler> logger) : base(logger) { }
 
-        public override void Handle(GameSession session, PacketReader packet) {
-            var type = (ChatType) packet.ReadInt();
+        public override void Handle(GameSession session, PacketReader packet)
+        {
+            ChatType type = (ChatType)packet.ReadInt();
             string message = packet.ReadUnicodeString();
-            string recipient = packet.ReadUnicodeString();
+            packet.ReadUnicodeString(); // Recipient
```
Comment (AlanMorel) [inline MapleServer2/PacketHandlers/Game/UserChatHandler.cs:20]: Again why remove the variable

**Review by AlanMorel — CHANGES_REQUESTED** [review]

File: MapleServer2/PacketHandlers/Game/StateSkillHandler.cs (hunk 1 above)
Comment (Sparkymod) [reply MapleServer2/PacketHandlers/Game/StateSkillHandler.cs:20]: Removing variable that is not being used, but add the comment to keep track for what it was using it in further implementations.

File: MapleServer2/Types/FieldState.cs (hunk 2 above)
Comment (Sparkymod) [reply MapleServer2/Types/FieldState.cs:67]: The "Mob" class and all his implementation, was something i made for testing before, to see if his behavior was the same as NPC, so removing it from the source would be better to not confuse people that was implemented.

File: MapleServer2/Types/Mob.cs (hunk 3 above)
Comment (Sparkymod) [reply MapleServer2/Types/Mob.cs:19]: Again, the mob was made for testing purpose, and is not sure that is the same as NPC

File: MapleServer2/PacketHandlers/Game/UserChatHandler.cs (hunk 4 above)
Comment (Sparkymod) [reply MapleServer2/PacketHandlers/Game/UserChatHandler.cs:20]: The variable is not being used, so for not lose track, i added as comment to remember what it was

File: MapleServer2/Types/Mob.cs (hunk 3 above)
Comment (AlanMorel) [reply MapleServer2/Types/Mob.cs:19]: When I implemented mob spawning on my end, I used this existing implementation as the base, so this is code worth keeping and later improving once the emulator gets there

**Comment by newgithubaccountlol** [comment]
I don't have any big comments for this PR but I would probably do 1 formatting PR (only formatting) and then another PR for changes like cleanup. With a huge change like this I don't even want to look at the cleanup changes

**Comment by AlanMorel** [comment]
> I don't have any big comments for this PR but I would probably do 1 formatting PR (only formatting) and then another PR for changes like cleanup. With a huge change like this I don't even want to look at the cleanup changes

Completely agree, I still went through every file but formatting is just noise and should have been done separately from clean up so the clean up can actually be reviewable on its own
